python/doc/examples/graphics/graphics_basics.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Creating graphics\n",
+    "# A quick start guide to graphics \n",
     "\n",
     "In this example, we show how to create graphics. We show how to create graphics and configure its axes and its colors. We show how to create a plot based on the combination of several plots."
```

Comment:
This comment does not really concern this line, but there is a typo in a sentence below:

We sometimes want to customize the graphics by chosing the type of point
chosing -> choosing

Further, there is a weird sentence below with a part in French:
I would rather get a dashed curve Je préfèrerai une courbe en pointillés : let us search for the available line styles.

Next (notice the typo even if you discard this suggestion):
In order to use the Curve class, it will be easier if we have a method to generate a Sample containing points **regularily** spaced in an interval.
Using the Curve class is easier  with a method that can generate a Sample containing regularly spaced points in an interval.

Next:
In some situations, we want to create curves which have different colors.
In some situations, we want to create curves with different colors.

Reply:
Excellent. Fixed all of them.